kong/include/kong/model/negotiation.proto

```diff
@@ -0,0 +1,47 @@
+syntax = "proto3";
+
+option go_package = "github.com/kong/koko/internal/gen/wrpc/kong/model;model";
+
+package kong.model;
+
+import "google/protobuf/struct.proto";
+
+message NegotiateServicesRequest {
+  DPNodeDescription node = 1;
+  repeated ServiceRequest services_requested = 2;
+  google.protobuf.Struct metadata = 3;
+}
+
+message DPNodeDescription {
+  string id = 1;
```

Comment:
The id should be already known from URI parameters. Do we need to include it here?